src/client/mixins/lead-aware.ts

```diff
@@ -21,4 +21,11 @@ export class LeadAwareMixin {
       { query: { _method: 'DELETE' } },
     );
   }
+
+  public async describeLeadFields() {
```

Comment:
- [ ] Recommend using the built-in `this.client.lead.describe()` rather than making the manual call.

Reply:
done